packages/js/src/utils/access-token.ts

```diff
@@ -47,7 +46,8 @@ export const decodeAccessToken = (accessToken: string): AccessTokenClaims => {
   const { 1: encodedPayload } = accessToken.split('.');
 
   if (!encodedPayload) {
-    throw new LogtoError('access_token.invalid_token');
+    // Opaque access token
```

Comment:
nit: We shouldn't assume it's an opaque token. You can say `non-JWT format token string` maybe?

Reply:
updated